.changeset/few-planets-brush.md

```diff
@@ -0,0 +1,5 @@
+---
+'slate-react': minor
+---
+
+Added renderPlaceholder prop to Editable component
```

Comment:
Can you change this to:

```
Added a `renderPlaceholder` prop to the `<Editable>` component for customizing how placeholders are rendered.
```
> Added a `renderPlaceholder` prop to the `<Editable>` component for customizing how placeholders are rendered.

Just because it will show up in the changelog so I try to keep it nicely written. Usually I just make edits to this on people's PRs by myself, but just putting this comment here since you're a maintainer so you know for the future.